addon.gypi

```diff
@@ -109,7 +109,7 @@
           '-luuid.lib',
           '-lodbc32.lib',
           '-lDelayImp.lib',
-          '-l"<(node_root_dir)/$(ConfigurationName)/<(node_lib_file)"'
+          '-l<(node_root_dir)/<(CONFIGURATION_NAME)/<(node_lib_file)'
```

Comment:
The process requires that `GYP(CONFIGURATION_NAME) === MsBuild(ConfigurationName)`
So now this resolution is a serpentius way, `GYP` sets it for `MsBuild`, and here `MsBuild` resolves it. And obviously it breaks for `ninja`/`win32`.